File: C++/pattern001.cpp
/* Pattern #001

.       * * * * *
.       * * * * *
.       * * * * *
.       * * * * *
.       * * * * *

*/

#include <iostream>

int main(void) {
    for (int i = 1; i <= 5; i++) {
        for (int j = 1; j <= 5; j++) {
            std::cout << "* ";
        }
        std::cout << std::endl;
    }
    return 0;
}

File: JavaScript/pattern001.js
/* Pattern #001

.       * * * * *
.       * * * * *
.       * * * * *
.       * * * * *
.       * * * * *

*/

let pattern = ""

for (let i = 1; i <= 5; i++) {
    for (let j = 1; j <=5; j++) {
        pattern += "* "
    }
    pattern += "\n"
}
console.log(pattern)